mlir/lib/Target/LLVMIR/Dialect/OpenMP/OpenMPToLLVMIRTranslation.cpp

```diff
@@ -1356,6 +1356,38 @@ class OmpParallelOpConversionManager {
   unsigned privateArgEndIdx;
 };
 
+namespace {
+omp::PrivateClauseOp findPrivatizer(Operation *from, SymbolRefAttr symbolName) {
```

Comment:
Nit: MLIR prefers static functions over functions in anonymous namespaces. Also, please comment functions.